detox/src/devices/Fbsimctl.js

```diff
@@ -173,6 +173,14 @@ class Fbsimctl {
     await this._execFbsimctlCommand(options);
   }
 
+  async resetContentAndSettings(udid) {
```

Comment:
Never wished for this title :( ... 
Let's talk a bit about the API:
1. I don't this is not a good API for reseting a permission, I much prefer if we can add an unset state to permissions with `appleSimUtils` https://github.com/wix/AppleSimulatorUtils/issues/8
2. Can there still can be a valid test case for resetting a device?
3. Since we aim Detox to be multi-platform framework I prefer not to give this method a name which is strongly recognized with Apple's terms, let's try thinking of something more generic maybe, like `reset` or `clearData` (not sure if these are better names). WDYT ?